packages/EasyCore/tests/Bridge/Laravel/Listeners/DoctrineRestartQueueOnEmCloseListenerTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Bridge\Laravel\Listeners;
+
+use Carbon\Carbon;
+use Doctrine\ORM\EntityManagerInterface;
+use EonX\EasyCore\Bridge\Laravel\Listeners\DoctrineRestartQueueOnEmCloseListener;
+use EonX\EasyCore\Tests\AbstractTestCase;
+use Exception;
+use Illuminate\Cache\Repository;
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Queue\Events\JobExceptionOccurred;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @covers \EonX\EasyCore\Bridge\Laravel\Listeners\DoctrineRestartQueueOnEmCloseListener
+ *
+ * @internal
+ */
+final class DoctrineRestartQueueOnEmCloseListenerTest extends AbstractTestCase
+{
+    /**
+     * Test `handle` succeeds.
+     *
+     * @return void
+     */
+    public function testHandleSucceeds(): void
```

Comment:
How about testing a case when EM is not closed?

Reply:
Done.